README.md

```diff
@@ -94,6 +102,8 @@ Since you may want to use the regex module to search for matches, `NA` also acts
 
 Handling missing/optional values in fields other than INFO or FORMAT can be done by checking for None, e.g `ID is not None`.
 
+Sometimes, multi-valued fields may contain missing values; in this case, the `without_na` function can be convenient, for example: `mean(without_na(FORMAT['DP'][s] for s in SAMPLES)) > 2.3`.
```

Comment:
Maybe the above-requested example invocation for `replace_na` can also go here, and you only quickly mention the actual signature of `replace_na(replacement=x)` above?

Reply:
Again, good point ;)